addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -361,6 +356,8 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend({
                 });
             } else {
                 self.$('.o_survey_session_close').click();
+                var descriptionDone = self.$('.o_survey_session_description_done').html();
+                self.$('.o_survey_session_manage_container').html("<h1>" + _t('Thank you!') + "</h1>" + descriptionDone);
```

Comment:
Why adding the description done in the manage_container ? It is currently its sibling, maybe you could simply add it directly into the container as hidden and toggle its visibility, instead of manipulating its content ? That way everything is already in-place, just hidden and we just have to carefully manage visibility.